README.md

```diff
@@ -54,6 +54,3 @@ If you need help using this app, we encourage you to:
 
 This project is maintained by Monalisa Octocat and friends. Use of this project under the [MIT License](LICENSE.md).
```

Comment:
A GitHub App built with probot that celebrates new user activity within your repository.